### PR TITLE
Fix multi-value resolve/reject, add TODO for login redirection on failures

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@jwforres review

File: assets/app/scripts/services/login.js [hunk 1]
```diff
@@ -51,11 +52,12 @@ angular.module('openshiftConsole')
         });
         if (debug) { console.log("RedirectLoginService.login(), redirecting", uri.toString()); }
         window.location.href = uri.toString();
+        // Return a promise we never intend to keep, because we're redirecting to another page
```
Comment (liggitt) [inline assets/app/scripts/services/login.js]: This is my favorite comment ever

**Comment by jwforres** [comment]
LGTM

**Comment by liggitt** [comment]
[merge]

**Pull request merged**